sql/core/src/test/scala/org/apache/spark/sql/CollationSQLExpressionsSuite.scala

```diff
@@ -1711,9 +1712,9 @@ class CollationSQLExpressionsSuite
 
   test("Support Mode.eval(buffer)") {
     case class UTF8StringModeTestCase[R](
-        collationId: String,
-        bufferValues: Map[UTF8String, Long],
-        result: R)
+      collationId: String,
+      bufferValues: Map[UTF8String, Long],
+      result: R)
```

Comment:
Revert it back, see https://github.com/databricks/scala-style-guide?tab=readme-ov-file#spacing-and-indentation